src/index.css

```diff
@@ -71,3 +71,33 @@ html {
 *::after {
   box-sizing: inherit;
 }
+
+.judge-columns {
```

Comment:
While we have some existing CSS in the in index.css file, we should try to avoid it moving forward. Instead of writing your own CSS to make a 3 column grid, we should use the `SimpleGrid` component [from Chakra UI](https://v2.chakra-ui.com/docs/components/simple-grid/usage).